tests/queries/0_stateless/02415_all_new_functions_must_be_documented.reference

```diff
@@ -568,6 +568,9 @@ randomPrintableASCII
 randomString
 randomStringUTF8
 range
+readWKBMultiPolygon
```

Comment:
This is not ok. All new functions should have documentation and version information